src/screens/WalletConnectApprovalSheet.js

```diff
@@ -374,96 +509,106 @@ export default function WalletConnectApprovalSheet() {
               >
-                <AvatarWrapper>
-                  {approvalAccountInfo.accountImage ? (
-                    <ImageAvatar
-                      image={approvalAccountInfo.accountImage}
-                      size="smaller"
-                    />
-                  ) : (
-                    <ContactAvatar
-                      color={
-                        isNaN(approvalAccountInfo.accountColor)
-                          ? colors.skeleton
-                          : approvalAccountInfo.accountColor
-                      }
-                      size="smaller"
-                      value={approvalAccountInfo.accountSymbol}
-                    />
-                  )}
-                </AvatarWrapper>
-                <Flex direction="row" flexShrink={1}>
-                  <LabelText>{approvalAccountInfo.accountLabel}</LabelText>
+                <Flex direction="row" align="center">
+                  <AvatarWrapper>
+                    {approvalAccountInfo.accountImage ? (
+                      <ImageAvatar
+                        image={approvalAccountInfo.accountImage}
+                        size="smaller"
+                      />
+                    ) : (
+                      <ContactAvatar
+                        color={
+                          isNaN(approvalAccountInfo.accountColor)
+                            ? colors.skeleton
+                            : approvalAccountInfo.accountColor
+                        }
+                        size="smaller"
+                        value={approvalAccountInfo.accountSymbol}
+                      />
+                    )}
+                  </AvatarWrapper>
+                  <LabelText position="relative">
+                    {approvalAccountInfo.accountLabel}
+                  </LabelText>
                 </Flex>
                 {type === WalletConnectApprovalSheetType.connect && (
                   <LabelText> 􀁰</LabelText>
                 )}
               </ButtonPressAnimation>
             </Column>
-            <Column>
-              <Flex justify="end">
-                <SwitchText align="right">
-                  {lang.t('wallet.network_title')}
-                </SwitchText>
-              </Flex>
-              <NetworkSwitcherParent
-                activeOpacity={0}
-                isMenuPrimaryAction
-                {...(android ? { onPress: onPressAndroid } : {})}
-                menuConfig={{
-                  menuItems,
-                  menuTitle: lang.t('walletconnect.available_networks'),
-                }}
-                onPressMenuItem={handleOnPressNetworksMenuItem}
-                useActionSheetFallback={false}
-                wrapNativeComponent={false}
-              >
-                <ButtonPressAnimation
-                  style={{
-                    alignItems: 'center',
-                    flexDirection: 'row',
-                    height: 38,
+            {isWalletConnectV2 ? null : (
```

Comment:
WC v1 session proposals show the wallet and network side-by-side. WC v2 has too much info, so they display as separate rows.